Client.Linq/Internal/Expressions/String/Contains.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace InfluxDB.Client.Linq.Internal.Expressions.String
+{
+    internal class Contains : IExpressionPart
```

Comment:
Please rename to `ContainsStr`
```suggestion
    internal class ContainsStr : IExpressionPart
```